airflow/providers/amazon/aws/waiters/appflow.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": 2,
+    "waiters": {
+        "run_complete": {
+            "operation": "DescribeFlowExecutionRecords",
+            "delay": 15,
+            "maxAttempts": 60,
+            "acceptors": [
+                {
+                    "expected": "Successful",
+                    "matcher": "path",
+                    "state": "success",
+                    "argument": "flowExecutions[?executionId=='<EXECUTION_ID>'].executionStatus"
```

Comment:
Just a question, why not to use Jinja for render templates instead of create own template mechanism?

Reply:
hmm that could be interesting yeah, I'll see how it could happen

Reply:
...but then as we handle the configuration for all the waiters at once, it'd fail/warn if params for other waiters are missing, which is bad. We'd have to systematically reconstruct a json for only the waiter we're going to use.

It'd be:

- json parse
- remove all waiters we're not interested in from ["waiters"]
- convert to string again, do jinja magic on string
- parse to json again

Reply:
I haven't really played with Jinja but that sounds like an interesting option.  

```
    json parse
    remove all waiters we're not interested in from ["waiters"]
    convert to string again, do jinja magic on string
    parse to json again
```

That sounds very inefficient :(

Reply:
ok I found a better solution, where I'm targeting only the acceptors' argument with jinja, so that I can restrict it to the waiter wanted without doing to much manipulations

Reply:
That sounds promising :+1: